packages/manager/modules/billing-components/src/components/services-actions/services-actions.html

```diff
@@ -107,7 +107,7 @@
         data-ng-if="(!$ctrl.service.shouldDeleteAtExpiration() || !$ctrl.service.isResiliated()) && !$ctrl.service.hasDebt() && !$ctrl.service.hasPendingResiliation()"
     >
         <oui-action-menu-item
-            ng-if="$ctrl.service.canBeResiliated($ctrl.user.nichandle) && (!$ctrl.service.hasPendingEngagement && !$ctrl.service.canCancelResiliationByEndRule() || $ctrl.service.canResiliateByEndRule())"
+            ng-if="$ctrl.service.canBeResiliated($ctrl.user.nichandle) || (!$ctrl.service.hasPendingEngagement && !$ctrl.service.canCancelResiliationByEndRule() || $ctrl.service.canResiliateByEndRule())"
```

Comment:
makes me wonder that "canBeResiliated" might be a wrong method name since it can be false and you can still go to resiliate screen

Reply:
You're right! Any suggestion around better naming for this case?
Since this ticket received a validation, I will merge the Pull Request.
Feel free to comment here. Thank you!